### ISSUE TITLE
Kube state metrics Information

### ISSUE THREAD

**Issue by hitesh1907nayyar** [helm/charts#13464]
Hi, 

I require information related to query in kube state metrics. 

Can somebody please help me related to finding below metrics through kube state metrics:

a) How to use kube_node_spec_unschedulable -Requirement-> I need to know how many pods are in unscheduled state .
b) How the calculation is done for kube_node_status_capacity_pods - Presently i am getting a sum of 220 from my two node cluster. how this value is coming in first place ?  Requirement ->   
I need to know the capacity of the nodes how many they can be created
c) What is this table showing me - kube_node_status_condition ?

Basically , i want to show a use case where when the number of unscheduled PODs increased then a new node spans up.
